### ISSUE TITLE
compilation error on new kernel 

### ISSUE THREAD

**Issue by nirmoy** [iovisor/bcc#2573]
Failed to compile BPF module on new kennel
``` 
python strlen_count.py
In file included from /virtual/main.c:2:
In file included from /lib/modules/5.4.0-rc4-1-default/build/include/uapi/linux/ptrace.h:142:
In file included from /lib/modules/5.4.0-rc4-1-default/build/arch/x86/include/asm/ptrace.h:5:
/lib/modules/5.4.0-rc4-1-default/build/arch/x86/include/asm/segment.h:254:2: error: expected '(' after 'asm'
        alternative_io ("lsl %[seg],%[p]",
        ^
/lib/modules/5.4.0-rc4-1-default/build/arch/x86/include/asm/alternative.h:240:2: note: expanded from macro 'alternative_io'
        asm_inline volatile (ALTERNATIVE(oldinstr, newinstr, feature)   \
        ^
/lib/modules/5.4.0-rc4-1-default/build/include/linux/compiler_types.h:210:24: note: expanded from macro 'asm_inline'
#define asm_inline asm __inline
                       ^
1 error generated.
Traceback (most recent call last):
  File "strlen_count.py", line 42, in <module>
    """)
  File "/usr/lib/python2.7/site-packages/bcc/__init__.py", line 347, in __init__
    raise Exception("Failed to compile BPF module %s" % (src_file or "<text>"))
Exception: Failed to compile BPF module <text>

```

**Comment by yonghong-song** [comment]
This should have been fixed by the following commit:
https://github.com/iovisor/bcc/commit/2d1497cde1cc9835f759a707b42dea83bee378b8
Redefine asm_inline for Linux 5.4+, Fixes #2546 (#2547) 

Could you try the latest trunk?

**Comment by nirmoy** [comment]
@yonghong-song, works from latest trunk . Thanks.
